consensus/consensus_v2.go

```diff
@@ -138,7 +138,21 @@ func (consensus *Consensus) HandleMessageUpdate(ctx context.Context, peer libp2p
 	return nil
 }
 
-func (consensus *Consensus) finalCommit(isLeader bool) {
+func (consensus *Consensus) finalCommit(waitTime time.Duration, viewID uint64, isLeader bool) {
+	consensus.getLogger().Info().Str("waitTime", waitTime.String()).
```

Comment:
possible race condition, but fixed in https://github.com/harmony-one/harmony/pull/4825

Reply:
minor